py/desisim/templates.py

```diff
@@ -340,10 +341,9 @@ def __init__(self, objtype='ELG', minwave=3600.0, maxwave=10000.0, cdelt=2.0,
             corresponding to BASEFLUX (Angstrom).
           basemeta (astropy.Table): Table of meta-data [nbase] for each base template.
           pixbound (numpy.ndarray): Pixel boundaries of BASEWAVE (Angstrom).
-          decamwise (speclite.filters instance): DECam2014-* and WISE2010-* FilterSequence
-          gfilt (speclite.filters instance): DECam2014 g-band FilterSequence
           rfilt (speclite.filters instance): DECam2014 r-band FilterSequence
-          zfilt (speclite.filters instance): DECam2014 z-band FilterSequence
+          normfilt (speclite.filters instance): FilterSequence of self.normfilter
+          decamwise (speclite.filters instance): DECam2014-* and WISE2010-* FilterSequence
```

Comment:
docstring seems out of since with actual `__init__` options: there doesn't appear to be a `decamwise` option,
`normfilt` vs. `normfilter`, and it is unclear to me if the `normfilter` option should be a string, a `FilterSequence` object, or either.